methods_and_flow_control/fizz_buzz.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+# FizzBuzz
+
+# Var
+(1..100).each do |number|
+  num_35 = ((number % 5 == 0) && (number % 3 == 0))
```

Comment:
You only use this variable in one place... You could inline it, i.e. make line 11 `if ((number % 5 == 0) && (number %3== 0))`.